Week1/w1_1.py

```diff
@@ -0,0 +1,12 @@
+def a(number):
+    if number <= 1:
+        return False
+    for i in range(2, int(number ** 0.5) + 1):

```

Comment:
can you explain me this for loop

Reply:
This calculates the square root of the given number using the exponentiation operator ** with 0.5 as the exponent